src/components/HomePage/LinksSection.jsx

```diff
@@ -31,32 +31,27 @@ const linkItems = [
     },
     {
         text: "CONTACT",
-        path: ""
+        path: "mailto:DIALUPSTUFF@GMAIL.COM"
```

Comment:
probably doesn't matter, but does it make sense to make the email lowercase? That's usually how I've seen mailtos

Reply:
yeah, probably. also not sure if this will have any impact but better safe than sorry 👍